app/src/main/java/org/oppia/app/player/content/ContentListFragmentPresenter.kt

```diff
@@ -0,0 +1,67 @@
+package org.oppia.app.player.content
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.oppia.app.application.ApplicationContext
+import org.oppia.app.databinding.ContentListFragmentBinding
+import org.oppia.data.backends.gae.model.GaeSubtitledHtml
+import javax.inject.Inject
+
+/** Presenter for [ContentListFragment]. */
+class ContentListFragmentPresenter @Inject constructor(
+  @ApplicationContext private val context: Context,
+  private val fragment: Fragment
+) {
+
+  private lateinit var binding: ContentListFragmentBinding
+  var contentCardAdapter: ContentCardAdapter? = null
+  var contentList: MutableList<GaeSubtitledHtml> = ArrayList()
+
+  fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
+
+    binding = ContentListFragmentBinding.inflate(inflater, container, /* attachToRoot= */ false)
+
+    binding.recyclerView.apply {
+
+      binding.recyclerView.layoutManager = LinearLayoutManager(context)
+      contentCardAdapter = ContentCardAdapter(context, contentList);
+      binding.contentCardAdapter = ContentCardAdapter(context, contentList);
+    }
+    fetchDummyExplorations()
+
+    return binding.root
+  }
+
+  //TODO (#121) :Replace this once interface for ExplorationDataController is available
+  private fun fetchDummyExplorations() {
+
+    contentList.add(
+      GaeSubtitledHtml(
```

Comment:
This seems like a suitable replacement to the fake fetcher above. Let's use this until we can use the domain controllers.

Reply:
okay done